### DISCUSSION TITLE
Request: Add weight value to schedules

### DISCUSSION THREAD

**Discussion by CypherPhyre** [Zrips/CMI#194] [Feature Requests]
I have an auto announcer scheduled for every 10 minutes when players are on. This announcer currently has 6 lines of text and is using subtitle placement. I also have a Server restart countdown using the subtitle placement. This one is triggered manually. If I trigger it while the auto announcer is active, or if the auto announcer starts to run after I trigger the countdown, the lines of text overlap and looks bad. If you could add a weight value to the schedule to override and/or prioritize when manually triggering them. That would be great.

**Comment by Zrips** [comment]
Can you provide those 2 schedulers for testing so we can be on same page?

**Comment by CypherPhyre** [comment]
Announcer1:
  Enabled: true
  MinPlayers: 1
  Delay: 600
  Repeat: true
  Randomize: false
  Commands:
  - subtitle! &l&aServer is in BETA
  - delay! 5
  - subtitle! &l&aNot all features are avaliable
  - delay! 5
  - subtitle! &l&aReport any bugs or issues
  - delay! 5
  - subtitle! &l&a/report <message>
  - delay! 5
  - subtitle! &l&aIf you need help, ask staff
#  - delay! 5
#  - subtitle! &eServer will stop in &610 &esec!
#  - delay! 5
#  - subtitle! &eServer will stop in &65 &esec!
#  - delay! 1
#  - subtitle! &eServer will stop in &64 &esec!
#  - delay! 1
#  - subtitle! &eServer will stop in &63 &esec!
#  - delay! 1
#  - subtitle! &eServer will stop in &62 &esec!
#  - delay! 1
#  - subtitle! &eServer will stop in &61 &esec!
#  - delay! 1
#  - kickall! &eServer will be back online soon!
#  - delay! 1
#  - stop

StopServer:
  Enabled: true
#  PerformOn:
#    1:
#      Hour: 3
#      Minute: 59
#      Second: 30
  Commands:
  - subtitle! &eServer will stop in &630 &esec!
  - delay! 5
  - subtitle! &eServer will stop in &625 &esec!
  - delay! 5
  - subtitle! &eServer will stop in &620 &esec!
  - delay! 5
  - subtitle! &eServer will stop in &615 &esec!
  - delay! 5
  - subtitle! &eServer will stop in &610 &esec!
  - delay! 5
  - subtitle! &eServer will stop in &65 &esec!
  - delay! 1
  - subtitle! &eServer will stop in &64 &esec!
  - delay! 1
  - subtitle! &eServer will stop in &63 &esec!
  - delay! 1
  - subtitle! &eServer will stop in &62 &esec!
  - delay! 1
  - subtitle! &eServer will stop in &61 &esec!
  - delay! 1
  - kickall! &eServer will be back online soon!
  - delay! 1
  - stop
